### ISSUE TITLE
CJK characters on windows

### ISSUE THREAD

**Issue by steelbluelee** [cosminadrianpopescu/vim-sql-workbench#20]
On windows using CJK codepage, sending characters are broken.
I am using this plugin through changing source code very little like the following.

in autoload/sw/server.vim -- line 229

`  let text = a:sql . "\n"`
  ==>
```
    if (exists('g:sw_windows_codepage'))
        let text = iconv(a:sql, &encoding, g:sw_windows_codepage) . "\n"
    else
        let text = a:sql . "\n"
    endif

```
and in vimrc file, the following line is added.
`
    let g:sw_windows_codepage="cp949"`

And, it works well without broken characters.

I think this change is too small to make a branch and issue a pull request.
Please apply this change to the source code.

Thx.

**Comment by cosminadrianpopescu** [comment]
I will include it in the next release.

**Comment by cosminadrianpopescu** [comment]
Fixed with 7.0
